README.md

```diff
@@ -1,30 +1,39 @@
+    3. Should show the amount paid on [taxes].
+  4. __List__ of all investments of a person.
 
 __NOTE:__ the implementation of an interface will not be evaluated.
 
-### Public Area
-1. __A public event list:__ When accessing the main route, the application should show a list of all the events registered, paging them every 10 events;
-- 1.1 The user should be able to filter the list of events by dates, or regions;
-2. __Event details:__ the application must allow the user to see the details of the event, by clicking on the event listing, or accessing the event link;
-3. __User signup:__ the application should allow the user to register by informing: Name, Email, Password, Bio, Profile Picture, City, and State;
-4. __User login:__ The application should allow the user to login using their credentials;
-- 4.1 The login should persist when the application is closed, or reloaded;
-
-### Logged Area
-5. __Friend invitation:__ the application will allow the user to enter an email to add as a friend;
-6. __Add as friend:__ The informed user should receive a friend request, or an invitation to register, if they are not already a user;
-7. __Friendship management:__ the user will be able to see your new friend requests, list your friends, and undo friendships;
-8. __Event registration:__ the application should allow the user to register an event by informing: Name, Description, Date, Time, and Place;
-- 8.1 The user should be able to edit and cancel events their events;
-9. __Invite friends to event:__ the user can invite their friends to events, being able to invite all friends, or only the selected ones;
-- 9.1 If the user has already been invited to the event, regardless of their status (confirmed, rejected, awaiting confirmation), the invited user should not be notified of the invitation again;
-10. __My event list:__ the user should be able to see their events, being able to filter them by those who will participate, and the ones that he created;
-11. __Manage event invitations:__ The user can accept, or reject, attend events.
-12. __Events management:__ The user can view their rejected events and undo rejections, deciding to participate, if the event has not yet occurred;
+### Interest Calculation
+
+The investment will pay 0.52% every month in the same day of the investment creation, if a withdraw is made than in that month no interest will be paid.
+
+Given that the interest is paid every month, than it should be treated as as compound interest, that means that interest that is not withdrawn will became part of the investment balance for the next interest payment.
+
+### Taxation
+
+Every time a withdrawn happens part of it will automatically be kept as taxes.
+The taxes apply only on the interest, that means that if the investment was created with 1000.00, and when now the balance was 1,100.00, if you withdraw 110.00 than the taxes will be calculated over 11.00 (10% = 1,100/1000).
```

Comment:
The taxes apply only to the interest, meaning that if ..., and now the balance is ..., ... then ...

Não ficou claro essa parte do 11.00

Reply:
a ideia eh dizer q a taxa se aplica apenas a uma parte do saque, tipo se na hora de sacar 10% do saldo (100) é de juros pago, então a taxa vai ser aplicada sobre 10% do q estiver sacado, no caso se sacar 110 então 11 representa os 10% de lucro, q a taxa deve ser sobre esses 11.

essa parte eu tinha achado dificil de escrever, e pelo visto ainda ficou ruim...

Reply:
Sugestão de texto:

When money is withdrawn, tax is triggered. Taxes apply only to the interest portion of the money withdrawn. For example, if the initial investment was 1000.00, the current balance is 1200.00, and you withdraw 150.00, then the taxes will be applied to 16.67% = (200/1200) of the amount withdrawn. Meaning that you should apply tax on 25.00 (150 x 16,67%).